phoenix-core/src/main/java/org/apache/phoenix/schema/MetaDataClient.java

```diff
@@ -4302,6 +4302,7 @@ public MutationState alterIndex(AlterIndexStatement statement) throws SQLExcepti
             boolean changingPhoenixTableProperty= evaluateStmtProperties(metaProperties,metaPropertiesEvaluated,table,schemaName,tableName);
 
             PIndexState newIndexState = statement.getIndexState();
+            // TODO: Change this for PHOENIX-4703
```

Comment:
Is it a self-referencing jira? Are we planning to include new syntax in the same PR/jira?

Reply:
I'm guessing you can just revert MetaDataClient from this PR, @gokceni , since you deferred the SQL changes to a subsequent JIRA.